client/ayon_core/hosts/nuke/api/utils.py

```diff
@@ -142,3 +149,86 @@ def is_headless():
         bool: headless
     """
     return QtWidgets.QApplication.instance() is None
+
+
+def create_error_report(context):
+    """Create an error report based on the given pyblish context.
+
+    This function iterates through the results in the context and formats any
+    errors into a comprehensive error report.
+
+    Args:
+        context (dict): Pyblish context.
+
+    Returns:
+        tuple: A tuple containing a boolean indicating success and a string
+        representing the error message.
+    """
+
+    error_message = ""
+    success = True
+    for result in context.data["results"]:
+        if result["success"]:
+            continue
+
+        success = False
+
+        err = result["error"]
+        error_message += "\n"
+        error_message += err.formatted_traceback
+
+    return success, error_message
+
+
+def submit_headless_farm(node):
+    # Ensure code is executed in root context.
+    if nuke.root() == nuke.thisNode():
+        _submit_headless_farm(node)
+    else:
+        # If not in root context, move to the root context and then execute the
+        # code.
+        with nuke.root():
+            _submit_headless_farm(node)
+
+
+def _submit_headless_farm(node):
+    """Headless farm submission
+
+    This function prepares the context for farm submission, validates it,
+    extracts relevant data, copies the current workfile to a timestamped copy,
+    and submits the job to the farm.
+
+    Args:
+        node (Node): The node for which the farm submission is being made.
+    """
+
+    host = registered_host()
+    create_context = CreateContext(host)
+
+    # Ensure CreateInstance is enabled.
+    for instance in create_context.instances:
+        if node.name() != instance.transient_data["node"].name():
+            continue
+
+        instance.data["active"] = True
+
+    context = pyblish.api.Context()
+    context.data["create_context"] = create_context
```

Comment:
If I get it correctly, the instances are disabled during publishing, so it is not necessary to disable then in create context, and create context is auto-created during publishing, so it does not have to be handled here at all.
```suggestion
    context = pyblish.api.Context()
```

Reply:
This bit of code, is ensuring the instance is active in create context. We are not disabling any instances in create context.